.github/actions/npm-publish-package/action.yml

```diff
@@ -29,16 +29,38 @@ runs:
   steps:
     - uses: actions/setup-node@v3
       with:
-        node-version: 17
+        node-version: 18
+
+    - uses: pnpm/action-setup@v2
+      with:
+        version: 7
+
+    - name: Install action dependencies
+      if: ${{ inputs.release_type == 'beta' }}
+      run: pnpm install
+      working-directory: ./.github/actions/get-latest-npm-beta-branch-version
+      shell: bash
+
+    - uses: ./.github/actions/get-latest-npm-beta-branch-version
+      if: ${{ inputs.release_type == 'beta' }}
+      id: get-latest-npm-beta-branch-version
+      with:
+        package: ${{ inputs.package }}
+        branch_name: ${{ github.head_ref }}
+
+    - name: Write beta NPM package version
+      if: ${{ steps.get-latest-npm-beta-branch-version.outputs.package_version != '' }}

```

Comment:
no issue if step `get-latest-npm-beta-branch-version.outputs` is skipped ? (ex: stable version)